### PR TITLE
docs(_run/single): update readme

### PR DESCRIPTION
**Pull request by rsum1**
Updated _run/single readme

### CODE REVIEW

File: _run/single/README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ use `ovrclk/akash:latest` to use the default image.
 
 __t1__
 ```sh
-DOCKER_IMAGE=ovrclk/akash:latest make kind-configure-image
+DOCKER_IMAGE=ovrclk/akash:stable make kind-configure-image
```
Comment (hydrogen18) [inline _run/single/README.md:88]: I think we should update this to the ghcr.io package, right @adam?

Comment (boz) [reply _run/single/README.md:88]: Yeah, let's.  I'll merge my PR and do a release to `v0.11.0-rc0` too so that we can use `:latest` here too.

**Comment by hydrogen18** [comment]
OK @rsum1 thanks for making the updates. Can you go through and update the docker image references to `ghcr.io/ovrclk/akash:latest` ? There are some in `_docs/kustomize` as well in the deployment.yaml files. Once the release is fixed that container reference should work.

File: _run/single/README.md (hunk 1 above)
Comment (rsum1) [reply _run/single/README.md]: updated to use ghcr.io package

**Review by hydrogen18 — APPROVED** [review]

**Review by troian — APPROVED** [review]

**Review by hydrogen18 — APPROVED** [review]

**Pull request merged**